confd/templates/janitor.properties.tmpl

```diff
@@ -1,2 +1,191 @@
-# Disable Janitor Monkey
-simianarmy.janitor.enabled = false
+# see documentation at:
+# https://github.com/Netflix/SimianArmy/wiki/Configuration
+
+# By default Janitor Monkey wakes up every hour
+simianarmy.scheduler.frequency = {{getv "/simianarmy/scheduler/frequency" "1"}}
```

Comment:
I don't really see that as a problem myself.  It is the way they are configured in the source and simplifies the configuration.

I can remove those properties, so they are configured in one place only.